src/frequenz/sdk/timeseries/_resampling.py

```diff
@@ -466,6 +466,17 @@ async def resample(self, *, one_shot: bool = False) -> None:
                 break
 
     def _update_window_end(self, timer_timestamp: datetime) -> None:
+        timer_error_s = abs((timer_timestamp - self._window_end).total_seconds())
```

Comment:
It shouldn't, but I don't trust it and rather cover for that situation in case the timer implementation ever changes. We know the timer is imprecise, I prefer not to assume it is only for one side. But I can add a comment to make this clear :+1: